ui-spacetimechart/src/index.ts

```diff
@@ -3,3 +3,6 @@ import './styles/main.css';
 export * from './components/SpaceTimeChart';
 export * from './components/PathLayer';
 export * from './components/ConflictLayer';
+export * from './hooks/useCanvas';
+export * from './components/workSchedules';
```

Comment:
I'm not sure that we need a subFolder WorkSchedule.
Here you can expose WorkSchedule and PatternRect.
```suggestion
export * from './components/WorkSchedules';
export * from './components/PatternRect';
```

Reply:
alright done